src/successModal/SuccessModal.unit.tsx

```diff
@@ -27,35 +27,23 @@ describe('<SuccessModal>', () => {
   })
 
   it('Should be not visible if isOpen is set to false', () => {
-    expect(wrapper.find('.kirk-modal-dialog').exists()).toBe(false)
+    expect(wrapper.find('[data-test="success-modal"]').exists()).toBe(false)
```

Comment:
I find this pattern interesting, however it would be better to discuss this with the team before introducing it into the code.